example/test/peripheral_details/peripheral_details_bloc_test.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:blemulator_example/model/ble_peripheral.dart';
+import 'package:blemulator_example/peripheral_details/bloc.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mockito/mockito.dart';
+
+import '../mock/mocks.dart';
+import '../mock/sample_ble_peripheral.dart';
+
+void main() {
+  PeripheralDetailsBloc peripheralDetailsBloc;
+  MockBleAdapter bleAdapter;
+  MockBlePeripheralRepository blePeripheralRepository;
+  BlePeripheral peripheral;
+
+  setUp(() {
+    bleAdapter = MockBleAdapter();
+    blePeripheralRepository = MockBlePeripheralRepository();
+    peripheral = SampleBlePeripheral();
+
+    when(blePeripheralRepository.blePeripheral).thenReturn(peripheral);
+
+    peripheralDetailsBloc =
+        PeripheralDetailsBloc(bleAdapter, blePeripheralRepository);
+  });
+
+  tearDown(() {
+    peripheralDetailsBloc.close();
+    bleAdapter = null;
+    blePeripheralRepository = null;
+  });
+
+  test('initial state is correct', () {
+    expect(peripheralDetailsBloc.initialState.peripheral,
+        blePeripheralRepository.blePeripheral);
+  });
+
+  test('close does not emit new states', () {
```

Comment:
why it should not?

Reply:
Removed this test since in fact it was not 'testing' relevant functionality (there was no reason or hard condition to ensure no events were emitted after close).